apple/Sources/Sargon/Extensions/Methods/Profile/AuthorizedDapp+Wrap+Functions.swift

```diff
@@ -17,4 +17,12 @@ extension AuthorizedDapp {
 	public func jsonData() -> Data {
 		authorizedDappToJsonBytes(authorizedDapp: self)
 	}
+	
+	public mutating func showDeposits(_ show: Bool) {
+		preferences.deposits = show ? .visible : .hidden
+	}
+	
+	public var showDeposits: Bool {
```

Comment:
maybe follow the usual guideline of using `is` prefix.

Reply:
`isDepositsVisible`?

Reply:
something like that.